lib/neo_service.rb

```diff
@@ -0,0 +1,21 @@
+require 'faraday'
+require 'figaro'
+require 'pry'
+require 'json'
+# Load ENV vars via Figaro
+Figaro.application = Figaro::Application.new(environment: 'production', path: File.expand_path('../config/application.yml', __FILE__))
+Figaro.load
+
+class NeoService
+
+  def get_url(date)
+    conn = Faraday.new(
+      url: 'https://api.nasa.gov',
+      params: { start_date: date, api_key: ENV['nasa_api_key']}
+    )
+
+    asteroids_list_data = conn.get('/neo/rest/v1/feed')
+    parsed_asteroids_data = JSON.parse(asteroids_list_data.body, symbolize_names: true)[:near_earth_objects][:"#{date}"]
+    binding.pry
+  end 
+end 
```

Comment:
This was a failed attempt to move the API request to a different class. I do not know why it wasn't working for me.